config.toml

```diff
@@ -21,54 +21,55 @@ date = [":git", "lastmod", "date", "publishDate"]
 
```

Comment:
haha - reminds me of my BASIC programming days... leaving space for other lines :)

Reply:
Did you see the commit message :D

https://github.com/ContainerSolutions/runbooks/pull/64/commits/91d03ead5e23fa51ed3092c88526e9e6098f4aa7